docs/source/developer_guide/core/caching.rst

```diff
@@ -3,10 +3,30 @@ Caching: implementation details
 
 This section covers some details of the caching mechanism which are not discussed in the :ref:`user guide <caching>`. If you are developing a plugin and want to modify the caching behavior of your classes, we recommend you read :ref:`this section <caching_matches>` first.
 
+Controlling caching
+-------------------
+
+When developing calculation and data classes, there are some methods you can use to determine how the hash is created:
+
+* To ignore specific attributes, a ``Node`` subclass can have a ``_hash_ignored_attributes`` attribute. This is a list of attribute names which are ignored when creating the hash.
+* For calculations, the ``_hash_ignored_inputs`` attribute lists inputs that should be ignored when creating the hash.
+* To add things which should be considered in the hash, you can override the :meth:`_get_objects_to_hash <aiida.orm.nodes.Node._get_objects_to_hash>` method. Note that doing so overrides the behavior described above, so you should make sure to use the ``super()`` method.
```

Comment:
I think you can write these as :meth:`~aiida.orm.nodes.Node._get_objects_to_hash` (using the now hidden code markers), and the tilde will make sure only the last part is used as the link. But I guess either way is fine?

Reply:
I think so as well, was not introduced by me but I will fix it.